service/ChessBoardManager.h

```diff
@@ -0,0 +1,66 @@
+#ifndef CHESSBOARDMANAGER_H
+#define CHESSBOARDMANAGER_H
+
+#include "../model/piece/Rook.cpp"
+#include "../model/piece/Knight.cpp"
+#include "../model/piece/Bishop.cpp"
+#include "../model/piece/Queen.cpp"
+#include "../model/piece/King.cpp"
+#include "../model/piece/Pawn.cpp"
+#include "../model/board/ChessBoard.h"
+#include <iostream>
+#include <vector>
+
+using namespace std;
+
+class ChessBoardManager {
+private:
+    ChessBoard board;
+    vector<Piece*> whitePieces;
+    vector<Piece*> blackPieces;
+
+public:
+    void setPieces() {
+        whitePieces.clear();
+        blackPieces.clear();
+
+        whitePieces.push_back(new Rook(Color::WHITE));
```

Comment:
figularyn massiwini bashga bir klassa geçir. sebäbi ol CheesBoardManager-e degishli dal, meselem on üçin PieceManager klass doredip bilyan. olam sana figuraly doredip berya we massivi berya. son senem ony CheesBoardManagere berip bilyan setter-in komegi bilen

Reply:
men immeny shu logikany diyyan: whitePieces.push_back(new Rook(Color::WHITE));